### ISSUE TITLE
FMC: Request for 3 "minor" quality of life improvements

### ISSUE THREAD

**Issue by oMrSmith** [mSparks43/747-400#490]
1. Allow entering the REF AIRPORT.
I think this should also work with engines started...

![747-400 - 2021-04-13 14 29 47](https://user-images.githubusercontent.com/78870770/114552893-5795d800-9c65-11eb-9c76-4ed9b9867705.png)

2. Allow to copy flap setting into scratch pad

![747-400 - 2021-04-13 14 29 47](https://user-images.githubusercontent.com/78870770/114552952-6a101180-9c65-11eb-8e94-b594c432ca8b.png)

3. Entering the RW in the RTE page should preselect it on DEP page:

![747-400 - 2021-04-13 14 28 18](https://user-images.githubusercontent.com/78870770/114553155-a2afeb00-9c65-11eb-9eb1-0ec328cd0db7.png)

**Comment by oMrSmith** [comment]
Thanks for implementing number 2. 
It was the most important of the three.

**Comment by crazytimtimtim** [comment]
Number 1 is because you have the IRS aligned afaik. Are you sure you can enter it once the IRS is aligned?

**Comment by oMrSmith** [comment]
I think entering the REF airport 
with engines startet,
yet _still on gound_ (before lift off)
works at least with every other 
boing add on I know.
I havn't tested yet, to confirm.
And: It's not a big issue anyway.

**Comment by oMrSmith** [comment]
I think entering the REF airport
with engines startet,
yet **_still on gound_** (before lift off)
works with every other
Boeing add on I know.
I've just tested it with the Zibo Mod.

**Comment by crazytimtimtim** [comment]
I agree with @oMrSmith actually, as the FCOM would've specified otherwise. I'll include this in my next PR.
